libs/design/src/molecules/feature/feature-icon/feature-icon.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component ({
+  selector: '[daff-feature-icon]',
+  template: '<ng-content></ng-content>',
+  host: {'class': 'daff-feature__icon'}
```

Comment:
changeDetection: ChangeDetectionStrategy.OnPush

Reply:
Resolved in 688add3